src/components/PostDetails/PostDetails.tsx

```diff
@@ -0,0 +1,160 @@
+import React, { useEffect, useState } from 'react';
+import { Post } from '../../types/Post';
+import { Loader } from '../Loader';
+import { NewCommentForm } from '../NewCommentForm/NewCommentForm';
+import { Comment, CommentData } from '../../types/Comment';
+import {
+  addComment,
+  getCommentsByPostId,
+  removeComment,
+} from '../../api/loadData';
+
+type Props = {
+  post: Post | null,
+};
```

Comment:
if you don't have post it's better to not render this component, PostDetails should be visible only when we have post